web/web/models/home_hub.py

```diff
@@ -49,7 +52,6 @@ def __repr__(self):
                                         backref=db.backref('service_location'),
                                         uselist=False)
```

Comment:
can we move the service location backref out of here as well?

Reply:
Yes this can be moved to service_location model